src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -125,6 +125,27 @@
                       Margin="{StaticResource XSmallTopMargin}"
                       IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"/>
 
+            <TextBlock x:Uid="FancyZones_OverlappingZonesLabel"
+                       Margin="{StaticResource SmallTopMargin}"
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
+
+            <RadioButton x:Uid="FancyZones_OverlappingZonesSmallest"
```

Comment:
Since the FancyZones Settings page is quite extensive, and thus save space and improve visibility, I'd go for a ComboBox instead of a group of RadioButtons.

Reply:
I agree, anything that has more than 2 choices, should be a combobox.

Reply:
Can we stick to Microsoft's reccommendations please, in stead of personal opinions?

> ## Is this the right control?
> 
> - Use a drop-down list to let users select a single value from a set of items that can be adequately represented with single lines of text.
> - Use a list or grid view instead of a combo box to display items that contain multiple lines of text or images.
> - _When there are fewer than five items, consider using radio buttons (if only one item can be selected) or check boxes (if multiple items can be selected)._
> - Use a combo box when the selection items are of secondary importance in the flow of your app. If the default option is recommended for most users in most situations, showing all the items by using a list view might draw more attention to the options than necessary. You can save space and minimize distraction by using a combo box.

https://docs.microsoft.com/nl-nl/windows/uwp/design/controls-and-patterns/combo-box

Reply:
Based on this, looks like Combo box is the way to go.

Reply:
@jay-o-way I am aware of those guidelines - but that's what they are, _guide_lines. If you look at the Fancy Zones settings page in general, it has a lot of options. Therefore using a ComboBox here can be appropriate to improve visual hierarchy and reduces the amount of scrolling that is needed to get to the bottom of the page. It's also a 'secondary' action so, in my opinion, not super important to explicitly mention all items with radiobuttons.


For example, in Windows 10 display settings the designers decided to use ComboBoxes as well for settings that have less then 5 items. I guess for the same reason (or the possibility that there might be more possible options that are device independent):
![image](https://user-images.githubusercontent.com/9866362/109162224-4148c100-7778-11eb-9b7a-3753d977aa56.png)

Reply:
@Jay-o-Way
consider the context of my suggestion, we are talking about the FZ settings, not deciding general rules valid everywhere.
For this specific settings, it's more important to add a very well explained section in the online documentation and we might add a link to it rather than trying to explain everything in radio button and fail.